zone/entity.cpp

```diff
@@ -2210,6 +2210,46 @@ Raid *EntityList::GetRaidByClient(Client* client)
 
 	return nullptr;
 }
+Raid* EntityList::GetRaidByBotName(const char* name)
+{
+
+	std::list<Raid*>::iterator iterator;
+	iterator = raid_list.begin();
+
+	while (iterator != raid_list.end()) {
```

Comment:
`for (const auto& r : raid_list)` maybe?

Reply:
Would keep from having to increment iterator.

Reply:
Went a different way.  Used a lamba to pull out just those members who have a name instead of having to iterator over all 72 members.  Then parse out if a name matches over the shorter list.  Not sure if there is any cost advantage though.  Thoughts?